client/db.go

```diff
@@ -0,0 +1,461 @@
+package client
+
+import (
+	"encoding"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/testutils"
+	gogoproto "github.com/gogo/protobuf/proto"
+)
+
+// Timestamp ...
+type Timestamp struct {
+	WallTime int64
```

Comment:
I'd strip out logical time from end client timestamp. It's utterly worthless from client perspective, or worse, confusing.